backend/backend/core/chains.py

```diff
@@ -7,15 +6,15 @@
 from .models import ModelRegistry
 from .tools import ToolRegistry
 
-from pydantic import BaseModel
-from typing_extensions import List
+from pydantic import BaseModel, Field
+from typing_extensions import List, Optional
 
 
 class JokeChain(object):
-    def __init__(self, model):
+    def __init__(self, models):
```

Comment:
The `models` parameter has been changed from a single model to a list of models.